spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/BaseBatchReader.java

```diff
@@ -45,10 +47,12 @@ abstract class BaseBatchReader<T extends ScanTask> extends BaseReader<ColumnarBa
       Schema expectedSchema,
       boolean caseSensitive,
       ParquetBatchReadConf parquetConf,
-      OrcBatchReadConf orcConf) {
+      OrcBatchReadConf orcConf,
+      VortexBatchReadConf vortexConf) {
```

Comment:
For the record: I prefer to put the batch size back to a common builder attribute, like `batchSize`.
Kept it this way, so we change only the minimally required code, but I would like to fix this later.